src/main/java/com/xpanxion/java/springboot/da1/demo/controller/student1/BookController1.java

```diff
@@ -47,6 +53,29 @@ public Book getBook(@PathVariable("id") Integer id) {
             return bookList;
         }
 
+        public void insertBook(Book newBook) {
+            jdbcTemplate.update(INSERT_BOOK,
+                    newBook.getTitle(),
+                    newBook.getIsbn(),
+                    newBook.getPrice(),
+                    0);
+        }
+
+        public void addToBookstore(Book newBook, int bookstoreId) {
+            List <Book> bookList;
+            bookList = jdbcTemplate.query(SELECT_BOOK_BY_NAME, (row, column) -> {
```

Comment:
There is a more efficient way of doing this using the `NamedParameterJdbcTemplate`.  I will merge this PR, but let's try to avoid an extra call to the database and try something like this:  https://www.devglan.com/spring-jdbc/fetch-auto-generated-primary-key-value-after-insert-spring-jdbc

Reply:
Gotcha! Thanks I will make those changes